tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -11,14 +11,81 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from cStringIO import StringIO
+
+import datetime
+import jinja2
 import json
+import os
+import yaml
+
+from cStringIO import StringIO
+from dateutil import parser
+from dateutil.tz import gettz
 
 
-def format_struct(struct):
-    io = StringIO()
-    json.dump(struct, io, indent=2)
-    return io.getvalue()
+def get_jinja_env():
```

Comment:
Since Sns is it's own class, we take the couple functions email an sns share (jinja_render, get_subject, and get_resource_tag_targets) and throw them into utils.